nx-dev/nx-dev/redirect-rules.js

```diff
@@ -192,6 +192,29 @@ const guideUrls = {
     '/cypress/generators/migrate-to-cypress-11',
 };
 
+const langSpecificGuideUrls = ['/l/r', '/l/a', '/l/n'].reduce(
+  (acc, cur) => ({
+    ...acc,
+    [`${cur}/core-concepts/configuration`]: '/reference/project-configuration',
```

Comment:
why do we need the language specific URL redirects?

Reply:
We might not need them since we have:
```
'/(l|latest|p|previous)/(a|angular|r|react|n|node)/:path*': '/:path*',
```

Let me test that quickly